time_monkey/settings/defaults.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for time_monkey project.
+
+Generated by 'django-admin startproject' using Django 1.11.12.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
```

Comment:
Also delete this doc string

Reply:
In it's entirety or just lines highlighted above?